### PR TITLE
Issue 211 (Add Kosaraju Algorithm)

### PR DESCRIPTION
**Pull request by sarthak17jain**
Fixed issue #211  (Add Kosaraju Algorithm)
Updated Graph.hpp by adding kosaraju() function.
@ZigRazor please review the changes and accept the pull request.

### CODE REVIEW

File: include/Graph/Graph.hpp [hunk 1]
```diff
@@ -355,6 +356,19 @@ namespace CXXGRAPH
 		* @param start Node from where traversing starts
 		* @return a vector of nodes that belong to C but not to M.
 		*/
+
+		virtual std::vector<std::vector<Node<T>>> kosaraju() const;
+		/**
```
Comment (ZigRazor) [inline include/Graph/Graph.hpp:361]: Please introduce the Doxygen documentation before the function declaration, and pay attention to not mess up the documentation of other method/function

**Review by ZigRazor — CHANGES_REQUESTED** [review]
Correct the comments done and then i can approve it

File: include/Graph/Graph.hpp (hunk 1 above)
Comment (sarthak17jain) [reply include/Graph/Graph.hpp]: Made the requested changes.
Corrected the commenting.
Moved the Doxygen documentation before the function declaration and corrected the documentation of the other function that I messed up earlier.

**Review by ZigRazor — APPROVED** [review]
Good

**Comment by ZigRazor** [comment]
This PR Fix #211

**Pull request merged**